docs/V7-Migration-Guide.md

```diff
@@ -382,185 +427,274 @@ The `Header` no longer auto-collapses action items into a `SlidePane`. Instead,
 </Header>
 ```
 
+Latest example can be found at [widgets.dojo.io/#widget/header/overview](https://widgets.dojo.io/#widget/header/overview)
 
-### label
+---
+
+### Label
 
 #### Property changes
 ##### Removed properties
 - `invalid?: boolean`
 	- replaced by `valid?: boolean`
 	- inverted validation API for consistency
+
 #### Example of migration from v6 to v7
 
-Latest example can be found on [widgets.dojo.io/#widget/label/overview](https://widgets.dojo.io/#widget/label/overview)
+##### v6 Example
+```tsx
+<Label invalid={true}>Invalid Label</Label>
+```
+
+##### v7 Example
+```tsx
+<Label valid={false}>Invalid Label</Label>
+```
+
+Latest example can be found at [widgets.dojo.io/#widget/label/overview](https://widgets.dojo.io/#widget/label/overview)
+
+---
 
+### List
```

Comment:
List content needs to be filled out once all changes to that widget are complete.